ghost/core/core/server/services/slack-notifications/service.js

```diff
@@ -0,0 +1,52 @@
+const DomainEvents = require('@tryghost/domain-events');
+const config = require('../../../shared/config');
+const labs = require('../../../shared/labs');
+const logging = require('@tryghost/logging');
+
+class SlackNotificationsServiceWrapper {
+    /** @type {import('@tryghost/slack-notifications/lib/SlackNotificationsService')} */
+    #api;
+
+    static create({siteUrl, isEnabled, webhookUrl}) {
```

Comment:
Will add JSDoc here. Forgot!

Reply:
@allouis not sure if `@returns {object} SlackNotificationsService` this is correct 😬

Reply:
Just `@returns {SlackNotificationsService}` ☺️